library/default/project/config/base_osw_fixed.yaml

```diff
@@ -0,0 +1,15 @@
+name: base_fixed_bottom_2024
+weather: era5_40.0N_72.5W_1990_2020.csv
+service_equipment:
+  - [ctv.yaml, 3]
+  - cab.yaml
+  - hlv.yaml
+layout: base_osw_fixed.csv
+inflation_rate: 0
+fixed_costs: fixed_costs_osw_fixed.yaml
+workday_start: 7
+workday_end: 19
+start_year: 2000
+end_year: 2020
+project_capacity: 600
+port: port_116km.yaml
```

Comment:
This isn't needed for an in situ repair, but the `port_distance` could be set to 116 from the port YAML's `site_distance`.